hotdoc/extensions/c/utils.py

```diff
@@ -82,7 +82,7 @@ def __create_macro_from_raw_text(self, raw, filename, filter_names, filenames):
         split = mcontent.split('(', 1)
         name = split[0]
 
-        if name.strip() in filter_names:
+        if name.strip() in filter_names and not self.app.database.get_comment(name.strip()):
```

Comment:
indent looks funny ?

Reply:
How so?

Reply:
hrm, I guess it's just the way it renders in my UI, the line is pretty long and wrapped around because I'm zoomed in